util/libcrypto.num

```diff
@@ -4912,3 +4912,6 @@ RSA_get0_pss_params                     ?	3_0_0	EXIST::FUNCTION:RSA
 X509_cmp_timeframe                      ?	3_0_0	EXIST::FUNCTION:
 OSSL_CMP_MSG_get0_header                ?	3_0_0	EXIST::FUNCTION:CMP
 BIO_f_prefix                            ?	3_0_0	EXIST::FUNCTION:
+X509_ALGOR_copy                         ?	3_0_0	EXIST::FUNCTION:
```

Comment:
We will need a separate PR for the 1.1.1 branch because the 3_0_0 needs to be replaced with 1_1_1e.

Reply:
So how is that done ?  I currently use the make command to (re)generate this file ? Should I check out that branch & regenerate ?

Reply:
Yes. You will get a conflict in this file when cherry-picking the patches on the 1.1.1 branch anyway.